fga-eps-mds/2020.1-stay-safe-front-end
# App Tutorial Screen

## Issue 
Resolves #106 


## Description

Adds the tutorial screen for guiding users on their first app usage.

## Screenshots (if appropriate):


![Screenshot_20201205-003837](https://user-images.githubusercontent.com/35741521/101258499-99d06b00-3701-11eb-8c6b-dc95314859da.png)

## Checklist:



## How to validate?

1 - Disable the light mode on app if necessary, **before running** it; (This is not a problem since the common users will see this screen only once with dark theme)
2 - Open the app and the navigate between slides;
3 - To see the tutorial again, uncomment the line 144 of *src/hooks/user.tsx* and save;
4 - Comment the line 144 and open the app again.
